java/src/main/java/ai/rapids/cudf/DType.java

```diff
@@ -54,11 +54,11 @@
    */
   TIMESTAMP_NANOSECONDS(8, 16, "timestamp[ns]"),
 
-  //DURATION_DAYS(4, 17, "NO IDEA"),q
-  //DURATION_SECONDS(8, 18, "NO IDEA"),
-  //DURATION_MILLISECONDS(8, 19, "NO IDEA"),
-  //DURATION_MICROSECONDS(8, 20, "NO IDEA"),
-  //DURATION_NANOSECONDS(8, 21, "NO IDEA"),
+  DURATION_DAYS(4, 17, "NO IDEA"),
```

Comment:
Do we want to leave the type string as `NO IDEA` here?  Thinking we should either use `int32`/`int64` or put an empty string here with a comment indicating there isn't (yet) a mapping of duration types to I/O type.